core/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/support/IncludeExclude.java

```diff
@@ -66,6 +66,7 @@
     public static final ParseField PATTERN_FIELD = new ParseField("pattern");
     public static final ParseField PARTITION_FIELD = new ParseField("partition");
     public static final ParseField NUM_PARTITIONS_FIELD = new ParseField("num_partitions");
+    private static final int hash_partitioning_seed = 31;
```

Comment:
can you use upper case for consistency and leave a comment about why we set the seed explicitly?